synapse/http/replicationagent.py

```diff
@@ -44,9 +44,11 @@ class ReplicationEndpointFactory:
     def __init__(
         self,
         reactor: ISynapseReactor,
+        instance_map: Dict[str, Any],
```

Comment:
I think this is incorrect?

```suggestion
        instance_map: Dict[str, InstanceLocationConfig],
```

(Same for the other places we pass this in now.)

Reply:
Fixed(as well as the other place and the imports involved) in [ffa74c4](https://github.com/matrix-org/synapse/pull/15578/commits/ffa74c4e2025870ff84941c7ebfdaca22c2754ed) and [cbafecc](https://github.com/matrix-org/synapse/pull/15578/commits/cbafecc9c2e6995dcf2a5cd799eeeae24dab59d0)